packages/react-aria-components/docs/react-aria-components.mdx

```diff
@@ -689,3 +689,23 @@ The documentation for each component includes many examples styled using vanilla
 </ExampleCard>
 
 </section>
+
+### Drag and Drop
```

Comment:
What section  (e.g. collections, date and time, forms, etc.) should DropZone and FileTrigger go under in the docs? I figured we might want to create a new section called drag and drop for DropZone but not sure where FileTrigger should go.